src/libraries/System.Memory/tests/ArrayBufferWriter/ArrayBufferWriterTests.Byte.cs

```diff
@@ -26,67 +26,132 @@ protected override void WriteData(IBufferWriter<byte> bufferWriter, int numBytes
         public void WriteAndCopyToStream()
         {
             var output = new ArrayBufferWriter<byte>();
-            WriteData(output, 100);
+            {
+                WriteData(output, 100);
+
+                using var memStream = new MemoryStream(100);
```

Comment:
nit: we don't need to use `var` here

```suggestion
                using MemoryStream memStream = new(100);
```